### ISSUE TITLE
Mini-canaloplasty

### ISSUE THREAD

**Issue by Sukramg** [monarch-initiative/MAxO#550]
# 137760 GLAUCOMA, PRIMARY OPEN ANGLE; POAG
https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2813598/

This method was introduced by Rękas et al. and involves a mini-incision technique for accessing the Schlemm’s canal without the need to prepare the classical TDM and having to close sutures. The main difference of this variation is the size of the flaps created, the superficial flap having a diameter of 4.0 x 1.5 mm, and the deep flap 1.0 x 1.0 mm. After locating Schlemm’s canal, the ostium can be probed with iTrack and an OVD is injected at every 2 hours on removal. The tensioning suture is placed in the same way as in conventional canaloplasty. Finally, without the removal of the deep scleral flap or dissection of the intrascleral lake or the TDM, the conjunctiva is closed with diathermy. As there is no intrascleral lake, the hypotensive effect will only be induced thorough the tensioning suture in the Schlemm’s canal.

**Comment by LCCarmody** [comment]
Is this a child of MAXO_0000959 canaloplasty?

**Comment by Sukramg** [comment]
Yes, this technique appears to be a variation of canaloplasty, which is represented by MAXO_0000959 in the Medical Actions Ontology (MAXO). It is essentially a modified canaloplasty procedure, with some key differences in the method of accessing Schlemm’s canal and the size of the flaps created. Therefore, it can be considered a 'child' or a specific type of the 'parent' procedure, canaloplasty.

**Comment by LCCarmody** [comment]
added
